pkg/addons/data/ConfigMap/openshift-branding/branding-config.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+data:
+  branding-config.yaml: |
+    apiVersion: branding.openshift.io/v1beta1
```

Comment:
I am confused by the existence of this file - are you looking to be updating `branding: osa` in pkg/addons/data/ConfigMap/openshift-console/console-config.yaml?  Presumably that will cover branding the admin console at the point an image is built which supports that config?

For the web console, I'd expect to see an edit of pkg/addons/data/ConfigMap/openshift-web-console/webconsole-config.yaml pointing extensions.stylesheetURLs to the Route fronting your httpd server